src/troubleshooting.rst

```diff
@@ -466,6 +471,66 @@ environments. If you encounter this issue in combination with containers,
 please let us know.
 
 
+Understanding Existing Workflows
+--------------------------------
+
+See what the Jinja2 is doing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sometimes understanding what Jinja2 templating is doing
+to a workflow configuration can be difficult.
+
+``cylc view --process`` (or ``-p``) shows you what the
+configuration will look like after Jinja2 processing.
+
+For example:
+
+.. code-block:: cylc-graph
+
+   {% for n in my_function(3) %}
+       R/1983/P{{n}}Y = cicada_{{n}}[-P{{n}}Y] => cicada_{{n}}
+   {% endfor %}
+
+is much easier to understand as:
+
+.. code-block:: cylc-graph
+
+      R/1983/P2Y = cicada_2[-P2Y] => cicada_2
+      R/1983/P3Y = cicada_3[-P3Y] => cicada_3
+      R/1983/P5Y = cicada_5[-P5Y] => cicada_5
+
+If you have installed and played a workflow
+(even using ``play --pause``) this processed content
+is shown in ``log/config/flow-processed.cylc``.
+
+Other Processed Configuration Files
```

Comment:
This bit can be merged into the existing section on log files which contains similar information.